lib/babel-istanbul-instrumenter.js

```diff
@@ -33,6 +32,13 @@ Instrumenter.prototype = Object.create(istanbul.Instrumenter.prototype);
 Instrumenter.prototype.constructor = Instrumenter;
 
 Instrumenter.prototype.instrumentSync = function(code, fileName) {
+  var plugins = this.babelOptions.plugins;
+  // If we're running in coverage, we're assuming that this is running in CI or in some
+  // form of test scenario and not being built for production. So it's fine to always
+  // force this plugin to work.
+  for (var plugin of this.plugins) {
```

Comment:
This is normally going to be a simple array, but I can do it just to be absolutely safe.